lib/evaluators/evaluatorHelpers/BaseExpressionEvaluator.ts

```diff
@@ -21,13 +22,16 @@ export interface ISharedContext {
    */
   enableExtendedXsdTypes?: boolean;
   functionArgumentsCache?: FunctionArgumentsCache;
+  // https://www.w3.org/TR/xpath-functions/#func-implicit-timezone (Redefined to reduce coupling to internal types)
```

Comment:
> (Redefined to reduce coupling to internal types)

I think it's fine to just use `ITimeZoneRepresentation`.
As long as we `export` this type form sparqlee, others can use it.

Reply:
I don't really understand what you mean here. Could you elaborate?

Reply:
`  defaultTimeZone?: { hours: number; minutes: number };` -> `  defaultTimeZone?: ITimeZoneRepresentation;`

Reply:
Oh, yes! But for the type export nothing special needs to happen right? It's not like I need to add something to our `index.ts`?

Reply:
You might need to, depends on what is already there.